src/components/App.js

```diff
@@ -7,7 +7,7 @@ import '../css/App.css';
 
 const stats = require('../stats.json');
 const years = Object.keys(stats).map(k => k.substring(0, 4)).reduce((p, c) => {
-  if (!p.includes(c)) p.push(c);
+  if (p.indexOf(c) === -1) p.push(c);
```

Comment:
It looks like core-js does polyfill [`String.includes`](https://github.com/zloirock/core-js/blob/master/es6/string.js#L7). 🤷‍♂️ `padStart` is part of the ES7 spec, and isn't polyfilled by default.